service/Dockerfile

```
@@ -0,0 +1,51 @@
+COPY . .
+
+RUN --mount=type=cache,target=/usr/local/cargo/registry \
+    --mount=type=cache,target=/usr/local/cargo/git \
+    --mount=type=cache,target=/app/target \
+    cargo build --release --locked \
+    && mkdir -p /app/bin \
+    && find target/release -maxdepth 1 -type f -executable -exec cp {} /app/bin/ \;
+
+# Final stage
+FROM debian:${DEBIAN_VERSION}-slim AS runtime
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)